lib/kitchen/directions/bake_index/v1.rb

```diff
@@ -1,142 +1,149 @@
-module Kitchen::Directions::BakeIndex
-  class V1
-    renderable
-
-    class Term
-      attr_reader :text
-      attr_reader :id
-      attr_reader :group_by
-      attr_reader :page_title
-
-      def initialize(text:, id:, group_by:, page_title:)
-        @text = text.strip
-        @id = id
-        @group_by = group_by
-        @page_title = page_title
-      end
-    end
+# frozen_string_literal: true
+
+module Kitchen
+  module Directions
+    module BakeIndex
```

Comment:
I know Rubocop says to do it this way instead of `Kitchen::Directions::BakeIndex` but I'm not a huge fan b/c it loses like 10 characters for so much indenting.  What about ignoring this rule at least in directions which have a lot of nesting?